packages/cpdf/cpdf.2.6/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "contact@coherentgraphics.co.uk"
+license: "Coherent Graphics Ltd Non-Commercial Use License Agreement"
```

Comment:
To make it pass the linter and comply to the SPDX format (see https://opam.ocaml.org/doc/Manual.html#opamfield-license)
```suggestion
license: "LicenseRef-Coherent-Graphics-Ltd-Non-Commercial-Use-License-Agreement"
```

Reply:
Done in trunk. Thanks.